### PR TITLE
feat(app): add google charts and update footer

### PR DESCRIPTION
**Pull request by scafer**

### CODE REVIEW

File: src/components/PricesChart/index.jsx [hunk 1]
```diff
@@ -1,65 +1,79 @@
+/* eslint-disable capitalized-comments */
```
Comment (DMCFA) [inline src/components/PricesChart/index.jsx:1]: Why not just remove the rule?

File: src/components/PricesChart/index.jsx [hunk 2]
```diff
@@ -1,65 +1,79 @@
...
+      textStyle: { color: isDarkMode ? 'white' : 'black' }
+    },
+    width: dimensions.width
+  };
+
+  /**
+   * Create Chart Data.
+   */
+
+  const createChartData = (prices) => {
```
Comment (DMCFA) [inline src/components/PricesChart/index.jsx:44]: Why leave commented out lines on the function?

File: src/services/utils/index.js [hunk 3]
```diff
@@ -40,6 +40,22 @@ export const getLastPrice = (product) => {
...
 };
 
+/**
+ * Get Avarage Price.
+ */
+
+export const getAveragePrice = (prices) => {
+  let sum = 0;
+
+  for (let it = 0; it < prices.length; it++) {
```
Comment (DMCFA) [inline src/services/utils/index.js:50]: Would something like this be a better approach to a for loop?

const sum = prices.map(price => parseFloat(getFormattedPrice(price))).reduce((acc, val) => acc + val, 0)

File: src/services/utils/index.js [hunk 4]
```diff
@@ -40,6 +40,22 @@ export const getLastPrice = (product) => {
   return 0;
 };
 
+/**
+ * Get Avarage Price.
```
Comment (DMCFA) [inline src/services/utils/index.js:44]: Small typo with avarage being average.

Following your convention on the function below this one, you could add:

 * @param {Array} prices - Array of prices.
 * @returns {string} - Average price rounded to two decimal places.
 */

**Review by DMCFA — CHANGES_REQUESTED** [review]

File: src/components/PricesChart/index.jsx (hunk 2 above)
Comment (scafer) [reply src/components/PricesChart/index.jsx:43]: Removed!

**Review by DMCFA — APPROVED** [review]

**Pull request merged**